lib/new_relic/agent/instrumentation/dynamodb/chain.rb

```diff
@@ -0,0 +1,34 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module Dynamodb::Chain
+    def self.instrument!
+      ::Aws::DynamoDB::Client.class_eval do
+        include NewRelic::Agent::Instrumentation::Dynamodb
+
+        %w[create_table
+          delete_item
+          delete_table
+          get_item
+          put_item
+          query
+          scan
+          update_item].each do |method_name|
```

Comment:
Since this array is the same as the one for `prepend`, what do you think about moving it to the `Instrumentation` module and setting it as a constant?

Reply:
hmm, yeah i see what you mean. the only thing that gives me pause to move it is that I like it being located here, because it makes it super clear exactly what methods are getting instrumented in the same place we normally would look to see what is getting instrumented.

Reply:
I get that. I think the convenience is really nice and makes this really readable. 

I'm worried about a situation where we need to add a new method in the future and only add it to prepend/forget about chain. Do you think that's likely? Hopefully testing in the CI would catch it.

Reply:
yeah i decided just to change it, if clarity is a problem in the future we can change something then

Reply:
Nice, thank you!